dbtwiz/commands/model.py

```diff
@@ -0,0 +1,42 @@
+import os
```

Comment:
`os`, `dedent` and `Console` are imported but never used.

Reply:
This is very WIP and will be rewritten/removed as we're working on generation stuff.